src/index.html

```diff
@@ -71,9 +76,12 @@
       </div>
     </div>
   </div>
-  <div class="footer-sticky-container" data-ng-if="(controllerName === 'SplashController')" data-ng-include="'footer.html'"></div>
+
+  <div class="footer-sticky-container" data-ng-if="(controllerName === 'SplashController')"
+    data-ng-include="'footer.html'"></div>
   <div data-ng-include="'acting_as.html'"></div>
   <div data-ng-include="'advisor_acting_as.html'"></div>
   <div data-ng-include="'delegate_acting_as.html'"></div>
 </body>
-</html>
+
+</html>
```

Comment:
@nkutub  - Can we add a newline at the end of the file. I know it's not a functional issue, but it's some sort of UNIX POSIX requirement for text files.

Reply:
I did add it not showing in gitHub